xbmc/cores/DataCacheCore.h

```diff
@@ -154,6 +156,7 @@ class CDataCacheCore
     int height;
     float fps;
     float dar;
+    bool isInterlaced;
```

Comment:
`m_` according to the code guidelines

Reply:
in a `struct` ? it is a new rule?

Reply:
https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#member-variables

Code guidelines mention member variables and don't specify a scope. So my interpretation is that struct member variables are also covered by this. I won't force it though.

Reply:
ok changed